Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -88,32 +89,32 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin,
   return file.ReadBytes(m_nand_keys.data(), NAND_KEYS_SIZE);
 }
 
-void NANDImporter::FindSuperblock()
+bool NANDImporter::FindSuperblock()
 {
   constexpr size_t NAND_SUPERBLOCK_START = 0x1fc00000;
-  constexpr size_t NAND_SUPERBLOCK_SIZE = 0x40000;
 
-  size_t superblock = 0;
-  u32 newest_version = 0;
-  for (size_t pos = NAND_SUPERBLOCK_START; pos < NAND_SIZE; pos += NAND_SUPERBLOCK_SIZE)
+  NANDSuperblock* superblock = reinterpret_cast<NANDSuperblock*>(&m_nand[NAND_SUPERBLOCK_START]);
```

Comment:
This is undefined behavior. Please use `std::memcpy` instead of `reinterpret_cast`.